applications/CompressiblePotentialFlowApplication/custom_processes/define_3d_wake_process.cpp

```diff
@@ -290,9 +290,9 @@ void Define3DWakeProcess::ComputeAndSaveLocalWakeNormal() const
 // of the wake surface in the wake direction. Note that the element size in span
 // direction is predetermined by the size of the conditions constituting the
 // trailing edge.
-void Define3DWakeProcess::ShedWakeSurfaceFromTheTrailingEdge() const
+void Define3DWakeProcess::ShedWakeSurfaceFromTheTrailingEdge()
 {
-    const Properties::Pointer pElemProp = mrStlWakeModelPart.pGetProperties(0);
```

Comment:
you could keep the const in the method and simply obtain a Properties::PointerToConst ? (or auto) no?

Reply:
Issue is when you follow the usage of pElemProp.
- It's used in `CreateWakeSurfaceNodesAndElements`
- Which uses it to call `CreateWakeSurfaceElements`
- Which uses it to call `ModelPart::CreateNewElement`, requiring a non-const version of the properties since I understand we want a reference to the same properties, not a copy.

Reply:
We do need a Properties::Pointer since, if you follow its usage, it ends up being passed to `ModelPart::CreateNewElement`.

However, since `pGetProperties` is not fixed by this PR, we can keep the `const` in the method.